### ISSUE TITLE
[question/request] output_metrics_options availability HTTP probe

### ISSUE THREAD

**Issue by jbkc85** [google/cloudprober#295]
To whom it may concern,

I was wanting to use `output_metrics_options` in the HTTP probe but apparently its only available in the external type.  Is there a reason for this?  I would love to be able to add additional tags to whatever probe type i can - specifically `environment` without having to write Prometheus regex labels.

Thanks!

ie:

```
  http_probe {
    protocol: HTTPS
    output_metrics_options {
      additional_labels: "environment=location1,service=example"
    }
  }
```

**Comment by manugarg** [comment]
@jbkc85 If you're looking for _output_metrics_options_ only for additional labels, there is another way to do it (though it's not released yet):

https://github.com/google/cloudprober/blob/ee717b051237150cd3a03825f033c789867887e3/probes/proto/config.proto#L100-L112

Though I understand that output_metrics_options can be useful for HTTP probes regardless in case you want to parse metrics from HTTP probe output.

**Comment by manugarg** [comment]
I just released a new version [v0.10.5](https://github.com/google/cloudprober/releases/tag/v0.10.5) that has support for the _additional_label_ field. Also added an [example](https://github.com/google/cloudprober/tree/master/examples/additional_label). Since this particular issue seems to be just about that, marking it closed. I've file a different feature request to build metrics from the HTTP probe output: #297.
